src/GlobalManager.sol

```diff
@@ -30,6 +30,11 @@ contract GlobalManager is XApp, Ownable {
      */
     uint256 public totalStake;
 
+    /**
+     * @dev Constant max gas to spend in call to delegate unstaking, ensuring success
+     */
+    uint64 private constant MAX_GAS_LIMIT = 200_000;
```

Comment:
I would name this something specific to the call we are applying it to.

Also, and you comment here saying what test cases showed gas to be.